v8.BUILD.bazel

```diff
@@ -69,7 +69,7 @@ cc_library(
   name = "core_lib",
   visibility = ["//visibility:public"],
   copts = COPTS + COPTS_TARGET_ARCH,
-  includes = ["include"],
+  includes = ["."],
```

Comment:
Wouldn't this expose all the directories to any rules that depend on `core_lib`?  Line 48 above adds `.` to the `COPTS` above so that it is available only to to this rule.